Jenkinsfile

```diff
@@ -9,6 +9,8 @@ pipeline {
         GIT_COMMIT_AUTHOR = sh(returnStdout: true, script:"git log --format='%an' -n 1 HEAD").trim()
         GIT_COMMIT_SUMMARY = "`<https://github.com/Kaggle/docker-python/commit/${GIT_COMMIT}|${GIT_COMMIT_SHORT}>` ${GIT_COMMIT_SUBJECT} - ${GIT_COMMIT_AUTHOR}"
         SLACK_CHANNEL = "#learnops"
+        KAGGLE_KEY = credentials('KAGGLE_API_KEY')
```

Comment:
Where/when is the environment variable set?

Reply:
In the jenkins web UI under the "credentials" section.